templates/install/ansible/playbooks/openshift/systemtests-dependecies.yml

```diff
@@ -0,0 +1,3 @@
+- hosts: localhost
+  roles:
+    - systemtests_dependencies
```

Comment:
Can you just push this into the systemtests.yml playbook?

Reply:
Sure that is not a problem.